src/modules/todolist.js

```diff
@@ -7,6 +7,7 @@ export default class ToDoList {
   static addTodo(todo) {
     const todos = ToDoList.getTodos();
     todos.push(todo);
+    console.log(todo);
```

Comment:
**Committing `Console.log()` is not a good practice. It makes your code ugly, and insecure, and affects performance.**
Refer here for more insight:
[JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)

Reply:
Thank you 🙂